templates/tableau-single-server-centos.template

```diff
@@ -279,7 +282,8 @@
                 "AutomatedInstaller": "https://s3-us-west-2.amazonaws.com/tableau-quickstart/automated-installer"
```

Comment:
This file should be in the this repo (under `scripts/`), not in the bucket it currently is.

Reply:
The reason I put this file in S3 bucket because it's similar to tableau-server.rpm and it is also a part of tableau server product. Do you still think this file should be in this repo?

Reply:
The litmus test really is: Do we expect that this will be fairly a fairly stable script that's used as part of the bootstrapping in the architecture or do we expect that this installer script will change with every single update of the installer bits?

My guess is that it's an installer bootstrapping script that should really be in the repo. I took a look at it, and it looks very similar to other shell scripts that we have worked on with partners for  bootstrapping.

Reply:
yes, it looks like an installer bootstrapping script, but it will change with every single update of the installer bits. Every time we release a tableau server, we will also release a corresponding automated installer. For example, this automated installer will be changed a lot in 2018Q1 compared 2017Q4.

Reply:
Ah, if that's the case then it would really be considered a part of the installer package/bits. Fine to leave it out of the repo...